package.json

```diff
@@ -2603,7 +2603,8 @@
         "clean": "rimraf ./out && rimraf ./tsconfig.tsbuildinfo && rimraf ./cljs-out",
         "compile-cljs": "npx shadow-cljs compile :calva-lib :test",
         "prewatch": "npm i && npm run clean && npm run update-grammar && npm run compile-cljs",
-        "watch": "npx tsc -watch -p ./tsconfig.json",
+        "watch": "npx tsc --watch --project ./tsconfig.json",
+        "watch-with-strict-nulls": "npx tsc --watch --project ./tsconfig.json --strictNullChecks",
```

Comment:
Might we want to just use `--strictNullChecks` on the normal watch? Or does that still throw many warnings/errors until we finish this work? When this work is finished, then we would add it to the normal watch?

Reply:
It has an incredible number of errors still. This is barely scratching the surface. I just wanted to get this kicked off and then work towards the end goal in sensible chunks, merging in frequently.

Reply:
I checked and there are [734 errors for this branch when I enable strictNullChecks](https://github.com/BetterThanTomorrow/calva/files/8192013/errors.txt) 😬

Reply:
I checked and there are 670 [errors](https://github.com/BetterThanTomorrow/calva/files/8192014/errors.txt) for this branch when I enable strictNullChecks 😬

(revised this comment since I was on the wrong branch)

Reply:
I'm now down to 333 in another branch I'm working on 😎